src/lib/stable_structures/stable_b_tree_map.ts

```diff
@@ -1,24 +1,24 @@
 import { None, Opt, Some } from '../candid/types/constructed/opt';
 import { nat64 } from '../candid/types/primitive/nats/nat64';
 import { nat8 } from '../candid/types/primitive/nats/nat8';
-import { encode, decode } from '../candid/serde';
+import { decode } from '../candid/serde';
 
 export interface Serializable {
     toBytes: (data: any) => Uint8Array;
     fromBytes: (bytes: Uint8Array) => any;
 }
 
 export function StableBTreeMap<Key = any, Value = any>(
-    keyType: Partial<Serializable>,
-    valueType: Partial<Serializable>,
+    keySerializable: Serializable,
+    valueSerializable: Serializable,
     memoryId: nat8
 ) {
     if (globalThis._azleIc !== undefined) {
         globalThis._azleIc.stableBTreeMapInit(memoryId);
     }
 
-    isSerializable(keyType);
-    isSerializable(valueType);
+    isSerializable(keySerializable);
+    isSerializable(valueSerializable);
```

Comment:
An opinionated naming preference thing here (i.e. feel free to ignore), `isSerializable` to me doesn't say it will throw or assert so I'd go with something more like `assertIsSerializable` or `ensureIsSerializable`. Just a thought.

Reply:
ah, figured out what it is. `isSerializable` to me says it will return a boolean. I think that's my issue. So I read it as you checking whether the thing was serializable, and then not doing anything with the result. So I think a different name wouldn't conflict with that convention.

Reply:
This is the convention in TypeScript for these if I'm not mistaken...just what I've always seen